lib/src/front_end/ast_node_visitor.dart

```diff
@@ -1840,12 +1841,21 @@ final class AstNodeVisitor extends ThrowingAstVisitor<void> with PieceFactory {
   @override
   void visitTypeParameter(TypeParameter node) {
     pieces.withMetadata(node.metadata, inlineMetadata: true, () {
-      pieces.token(node.name);
       if (node.bound case var bound?) {
```

Comment:
If only we had https://github.com/dart-lang/language/issues/2537 we could reduce indentation here.